src/ephysatlas/features.py

```diff
@@ -281,18 +281,17 @@ def dart_subtraction_numpy(data, fs, geometry, **params):
     )
 
     # Ensure scratch directory exists
-    #This is for SDSC calculations
+    # This is for SDSC calculations
     scratch_dir = Path("/scratch/prai1/dartsort/")
```

Comment:
This shouldn't be hard-coded no ? This may be either a parameter or a configuration.